apps/design-system/next.config.mjs

```diff
@@ -26,16 +26,27 @@ const nextConfig = {
     return [
       ...(BASE_PATH.length
         ? [
-            {
-              source: '/',
-              destination: BASE_PATH,
-              basePath: false,
-              permanent: false,
-            },
-          ]
+          {
+            source: '/',
+            destination: BASE_PATH,
+            basePath: false,
+            permanent: false,
+          },
+        ]
         : []),
     ]
   },
+  // Turbopack configuration to handle .md files with raw-loader
+  // This mirrors the webpack configuration added by withContentlayer
+  // and ensures both bundlers can process content files properly
+  turbopack: {
+    rules: {
+      '*.md': {
+        loaders: ['raw-loader'],
+        as: '*.js',
+      },
+    },
+  },
```

Comment:
As the comment implies: there were hiccups building the .mdx files. This fixes it.